robot/impl/local_robot_test.go

```diff
@@ -4058,7 +4058,7 @@ func newErrorSensor() sensor.Sensor {
 func newInvalidSensor() sensor.Sensor {
 	s := &inject.Sensor{}
 	s.ReadingsFunc = func(ctx context.Context, extra map[string]interface{}) (map[string]interface{}, error) {
-		return map[string]any{"ThatsMyWallet": 1, "ThatsNotMyWallet": 2}, nil
+		return map[string]any{"ThatsMyWallet": true, "ThatsNotMyWallet": false}, nil
```

Comment:
why did this test case change?

Reply:
This sensor was used in a test that assumed that returning 1 would lead to reconfiguring. Since that is no longer true it had to change to true